source/overview/security.rst

```diff
@@ -79,40 +138,45 @@ Security updates address newly discovered attacks reported to Mattermost, Inc. b
 
-- Mattermost offers HIPAA-relevant **Technincal Safeguards** including:
+Mattermost offers HIPAA-relevant *Technincal Safeguards* including:
 
-     - `Integrity Controls <https://docs.mattermost.com/overview/security.html#integrity-audit-controls>`_
-     - `Access Control <https://docs.mattermost.com/overview/security.html#access-control-policy>`_
-     - `Transmission Security <https://docs.mattermost.com/overview/security.html#transmission-security>`_
-     - `Audit Controls <https://docs.mattermost.com/overview/security.html#integrity-audit-controls>`_
+- Integrity Controls <https://docs.mattermost.com/overview/security.html#integrity-audit-controls>`_
+- Access Control <https://docs.mattermost.com/overview/security.html#access-control-policy>`_
+- Transmission Security <https://docs.mattermost.com/overview/security.html#transmission-security>`_
+- Audit Controls <https://docs.mattermost.com/overview/security.html#integrity-audit-controls>`_.
 
-- HIPAA-compliant deployments commonly consider the following:
+HIPAA-compliant deployments commonly omit the contents of messages from mobile push and email notifications:
 
-     - Omitting the contents of messages from mobile push and email notifications:
+- If your `Push Notifications Contents <https://docs.mattermost.com/administration/config-settings.html#push-notification-contents>`_ option is set to ``Send full message snippet`` there is a chance Personal Health Information (PHI) contained in messages could be displayed on a user's locked phone as a notification. 
+To avoid this, 
+set the option to ``Send generic description with user and channel names`` or ``Send generic description with only sender name``.
 
-        - If your `Push Notifications Contents <https://docs.mattermost.com/administration/config-settings.html#push-notification-contents>`_ option is set to ``Send full message snippet`` there is a chance Personal Health Information (PHI) contained in messages could be displayed on a user's locked phone as a notification. To avoid this, set the option to ``Send generic description with user and channel names`` or ``Send generic description with only sender name``.
-        - Similarly, setting `Email Notifications Contents <https://docs.mattermost.com/administration/config-settings.html#email-notification-contents>`_ to ``Send generic description with only sender name`` will only send the team name and name of the person who sent the message, with no information about channel name or message contents included in email notifications.
+- Setting `Email Notifications Contents <https://docs.mattermost.com/administration/config-settings.html#email-notification-contents>`_ to ``Send generic description with only sender name`` will send the team name and name of the person who sent the message absent information about channel name or message contents included in email notifications.
```

Comment:
should "included in email notifications" be removed

Reply:
your call. works either way to my sense....